### PR TITLE
Task/refactor

### PR DESCRIPTION
**Pull request by kazoompa**

### CODE REVIEW

File: tests/utils.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import sys
+from argparse import ArgumentParser
+from obiba_mica import MicaClient
+
+class Utils:
+  # SERVER = 'https://mica-demo.obiba.org'
```
Comment (ymarcon) [inline tests/utils.py]: Enable mica-demo by default so that Travis build can refer to something that exists.

**Review by ymarcon — CHANGES_REQUESTED** [review]
You should add functions documentation (see how done in obiba_opal). As an example of the python doc output use the command:

```
pydoc obiba_opal.analysis
```
(clearly the output could be better for the params by the way)

If you wish to do it in a future PR that's ok.

**Review by ymarcon — CHANGES_REQUESTED** [review]
You should add functions documentation (see how done in obiba_opal). As an example of the python doc output use the command:

```
pydoc obiba_opal.analysis
```
(clearly the output could be better for the params by the way)

If you wish to do it in a future PR that's ok.

File: tests/utils.py (hunk 1 above)
Comment (kazoompa) [reply tests/utils.py]: Surely a mistake, will do changes and will notify...

**Comment by kazoompa** [comment]
I will complete the rest of the documentation in another PR

**Review by ymarcon — APPROVED** [review]

**Pull request merged**